Add support for RML language

pie-catt opened the pull request:
Add support for RML language.
https://rmlatdibris.github.io/

RML is a rewriting-based and system agnostic Domain Specific Language for Runtime Verification which decouples monitoring from instrumentation by allowing users to write specifications and to synthesize monitors from them, independently of the System Under Scrutiny (SUS) and its instrumentation.

RML has been developed and is currently maintained by [DIBRIS](https://www.dibris.unige.it/en/) - University of Genoa

<img width="751" alt="RML_sample" src="https://user-images.githubusercontent.com/33451203/103168542-4d75d800-4834-11eb-8768-7bb30938ebbd.png">

pie-catt commented:
Hello @tancnle. Are there any updates on this? Thank you for your work

tancnle commented:
Thanks for the ping @pie-catt. I will have a look at it this week. Thank you for your patience. Really appreciated 🙇🏼

tancnle commented on lib/rouge/lexers/rml.rb:
```diff
@@ -0,0 +1,157 @@
+    module Lexers
+    class RML < RegexLexer
+      title "RML"
+      desc "A system agnostic domain specific language for runtime monitoring and verification"
```
**suggestion:** Could we add a URL to the language spec page? WDYT?

```suggestion
      desc "A system agnostic domain-specific language for runtime monitoring and verification (https://rmlatdibris.github.io/)"
```

tancnle commented on lib/rouge/lexers/rml.rb:
```diff
@@ -0,0 +1,157 @@
+# frozen_string_literal: true
+
+module Rouge
+    module Lexers
```
**suggestion:** We use 2-space indentation in our code base. Would it be possible to fix all the indentations on this file? 🙏🏼

tancnle reviewed (commented):
Thank you for your contribution @pie-catt ❤️ I have added some suggestions for your consideration. Please let me know what you think.

pie-catt commented:
Thank you so much for your support @tancnle 🙏🏼. I updated my pull request with your suggestions. I fixed code indentation, renamed variables names to snake-case, removed superfluous and closing comments. I also added RML language entry in [Languages.md](https://github.com/rouge-ruby/rouge/blob/master/docs/Languages.md) file. Thank you for your work

tancnle commented:
Thank you for considering the suggestions @pie-catt 🙇🏼 LGTM. Let's launch this into orbit 🚀
